src/Core/Context/CurrentContext.cs

```diff
@@ -321,6 +321,7 @@ public async Task<bool> AccessReports(Guid orgId)
                     && (o.Permissions?.AccessReports ?? false)) ?? false);
     }
 
+    [Obsolete("CreateNewCollections is deprecated, use the CollectionAuthorizationHandler for controller level permission checks or CurrentContext.GetOrganization if permission checks are needed outside of that scope.")]
     public async Task<bool> CreateNewCollections(Guid orgId)
```

Comment:
This looks unused. Let's delete it! (even better than marking it as obsolete)

Reply:
Deleted.